nornir/core/inventory.py

```diff
@@ -105,6 +105,18 @@ def __contains__(self, value: object) -> bool:
         else:
             return any([value == g for g in self])
 
+    def add(self, group: "Group") -> None:
+        # only add the group if it doesn't exist
+        if not self.__contains__(group):
+            self.append(group)
+
+    def remove(self, group: "Group") -> None:
+        if self.__contains__(group):
+            # remove overrides the parent class so calling super()
+            super(ParentGroups, self).remove(group)
```

Comment:
this might be confusing

Reply:
any reason to not use py3-style super; `super().remove(...)`?

Reply:
No good reason.  Will update.